### ISSUE TITLE
UCI datasets benchmark code

### ISSUE THREAD

**Issue by arashno** [bioinf-jku/SNNs#15]
Hi,
I couldn't find any code regrading the UCI 121 tasks benchmark mentioned in the paper in this repo.
Could you please provide that code as well?

Thanks

**Comment by VihaanAkshaay** [comment]
Yes, I have doubts regarding the same, I don't understand how the folds and validation folds files work.
